standards/out-of-hours.md

```diff
@@ -0,0 +1,60 @@
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
+
+Either kind of person, and all of those on the spectrum between, however, can easily burn out if a few core criteria are not met:
+
+- No one person is on call more than 50% of the time, and ideally no more than 30% of the time.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
+- The rota repeats regularly, so people know when they are on call over the coming weeks and months, and can plan accordingly.
+- People on call are appropriately paid for being available on call and for responding to incidents.
+- People on call know when they are expected to be available when on call, whether 24/7, 7am-10pm, or some other, agreed hours.
+- People on call can trade shifts with one another when they need to.
+- People on call for the first time are on call with someone experienced to being on call on their support rota, and ideally someone they know well.
+- People on call understand the circumstances under which they will be paged. This should ideally be a clearly defined list of alerts with agreed steps to resolve, but some services may not yet be in a position to support this.
+
+### Contacting people on call about an issue
+
+To allow people to manage how they are contacted when on call, the MOJ uses [PagerDuty](https://www.pagerduty.com). PagerDuty (and systems like it) allows people on call to specify how they prefer to be contacted (in-app notifications, phone calls, text messages) and when, relative to an incident being raised. It also ensures that the people on call are paged according to pre-defined escalation policies (primary first, then secondary, then repeat).
+
+People on call should *never* be contacted directly to report issues, especially not through their personal phone number, because this creates a culture in which specific people feel like they are on call 100% of the time, just in case a senior person contacts them.
```

Comment:
I hesitate to say this, but should there be some central audit trail of incidents? I'm used to an alert generating something like a ServiceNow ticket so that a full history of the incident and it's resolution is recorded. What are the requirements at MOJ in this regard? I imagine it would be good in many ways to have something in place, not least for management to be able to monitor/ report on overall incidents over a given time period?

Reply:
The ITSM-managed bits of MOJ do, in fact, use ServiceNow for this. PagerDuty also offers that sort of functionality, though, and most of the digital ops teams are already using it to one degree or other. My intention is that we make more, better use of PagerDuty for tracking all incidents for digital services.